web-app/js/smartR/d3Heatmap.js

```diff
@@ -336,7 +338,7 @@ SmartRHeatmap = (function(){
                     var rowValues = [];
                     for(var i = 0; i < significanceValues.length; i++) {
                         var significanceValue = significanceValues[i];
-                        rowValues.push([i, significanceValue]);
+                        rowValues.push([i, Math.abs(significanceValue)]);
```

Comment:
d.significance comes from here 
 
    var significanceIndexMap = jQuery.map(significanceValues, function(d, i) {
                return {significance: d, idx: i};

Where significanceValues is assigned from rowValues after sorting is done. This means that after single sorting all negative values will become positive. I am testing it right now